### DISCUSSION TITLE
Elsa V3 in isolated network

### DISCUSSION THREAD

**Discussion by robverhoef-united** [elsa-workflows/elsa-core#4085] [Ideas]
Hello,

I am an Elsa V2 user and very much a big fan! I am currently doing some tests with Elsa V3. I have a usecase where we use the Elsa designer in a dedicated and isolated part of our network without full access to the internet. With V2 thats working perfectly fine. With V3 the designer is not working. I  discovered that this is probably due to this ("_https://rsms.me/inter/inter.css"_) stylesheet that is used in the index.cshtml file. This stylesheet has references to some fonts which are also not loaded when i offline the inter.css file.

Would it be possible to offload these fonts into the Elsa package itself and make it a true standalone, no dependencies application?

Thanks, Rob

**Comment by sfmskywalker** [comment]
Certainly! If you open an issue for this I’ll make sure it gets added before the final 3.0 release.

**Reply by robverhoef-united** [reply]
Thanks for the reply, added the request in issue #4094

**Reply by sfmskywalker** [reply]
Thanks!
